utils/urlParams/index.js

```diff
@@ -0,0 +1,32 @@
+import { PAGE_HOME, PAGE_SPEED_UP } from '../../constants'
+
+export const persistKeyUrlParams = router => {
+  const keyParams = new Set([
+    'network'
+  ]);
+
+  const newParams = {}
+
+  const currParams = new URLSearchParams(router.query)
+  currParams.forEach((val, key) => {
+    if(keyParams.has(key)) {
+      newParams[key] = val
+    }
+  })
+
+  return new URLSearchParams(newParams)
```

Comment:
Why not use the native URLSearchParams prototype methods instead of writing our own helpers?

Reply:
Oh - i don't think there is a native one to do what we need here. forEach is a native method, but what I need to do here is to only copy properties that we decided _should_  get copied. In this case 'network' is the only one, but im not sure if we'll want others. We _don't_ want to copy the transactionId because we're going "back" to the dashboard (will change to cid).

Writing a helper like this seemed more correctly generic than to just delete the transactionId. In the future if/when more properties are added, that could lead to unexpected behavior I'd think.